code-of-conduct.md

```diff
@@ -0,0 +1,46 @@
+census-developers@googlegroups.com
+# OpenCensus Code of Conduct 
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers of OpenCensus pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation. People violating this code of conduct may be banned from the community and all its events.
+
+## Our Standards and Values
+
+- Be honest:
```

Comment:
why ends with semicolon?

Reply:
Typo.. its been removed till we have a section on honesty.